tensorflow/jobs/docker/2.13/py3/Dockerfile.gpu

```diff
@@ -36,8 +36,8 @@ RUN ${PIP} install --no-cache --upgrade \
         numpy==1.23.5 \
         openfermion==1.5.1 \
         pandas==2.0.0 \
-        PennyLane==0.31.0 \
-        PennyLane-Lightning==0.31.0 \
+        PennyLane==0.32.0 \
+        PennyLane-Lightning==0.32.0 \
```

Comment:
Should we also upgrade PL in base and pytorch to match?

Reply:
As this is an upgrade to the image we pull from, I am scoping this change to only include Tensorflow. There will be a follow up PR to raise those versions as well.